WeScan/ImageScannerController.swift

```diff
@@ -84,15 +84,12 @@ public final class ImageScannerController: UINavigationController {
             if #available(iOS 11.0, *) {
                 // Use the VisionRectangleDetector on iOS 11 to attempt to find a rectangle from the initial image.
                 VisionRectangleDetector.rectangle(forImage: ciImage) { (quad) in
-                    detectedQuad = quad
+                    detectedQuad = quad?.toCartesian(withHeight: image.size.height)
                     detectedQuad?.reorganize()
-
-                    let editViewController = EditScanViewController(image: image, quad: detectedQuad, rotateImage: false)
-                    self.setViewControllers([editViewController], animated: true)
```

Comment:
@Boris-Em do you know whether this is OK? it seems like a breaking change to me

Reply:
This code should not be removed as the `defer` block is *not* called - Vision uses escaping closures.